.github/workflows/automatic-changelog.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           git config --global user.email 'changelog-bot@opentripplanner.org'
 
       - name: Generate changelog entry from PR information
-        if: github.event.pull_request.merged
+        if: github.event.pull_request.merged && github.repository_owner == "opentripplanner"
```

Comment:
I think this should be applied to the `Checkout` and `Configure Git User` as well

Reply:
Yes, that would be a good name. I will add the if clause and rename the token to `CHANGELOG_TOKEN`.

Reply:
I have added the new token `CHANGELOG_TOKEN`.